apps/src/dance/Dance.js

```diff
@@ -599,6 +599,12 @@ Dance.prototype.onPuzzleComplete = function (result, message) {
     this.studioApp_.playAudio('failure');
   }
 
+  analyticsReporter.sendEvent(EVENTS.DANCE_PARTY_VALIDATION, {
+    result,
+    danceMessage,
```

Comment:
Can we send `message` instead of `danceMessage` so that the key is sent?  It feels like this will be more compact, and also more consistent, independent of the user's language.

Reply:
Yes, definitely. Thanks for the suggestion!